src/components/common/UserProfile/UserMeta.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { useMediaQuery } from 'react-responsive';
+
+import CopyableAddress from '~shared/CopyableAddress';
+import ExternalLink from '~shared/ExternalLink';
+import Heading from '~shared/Heading';
+import Icon from '~shared/Icon';
+import Link from '~shared/Link';
+import UserMention from '~shared/UserMention';
+import UserAvatar from '~shared/UserAvatar';
+
+import { stripProtocol } from '~utils/strings';
+import { useAppContext } from '~hooks';
+import { User } from '~types';
+
+import query from '~styles/queries.css';
+import styles from './UserMeta.css';
+
+const componentDisplayName = 'common.UserProfile.UserMeta';
+
+const MSG = defineMessages({
+  editProfileTitle: {
+    id: `${componentDisplayName}.editProfileTitle`,
+    defaultMessage: 'Edit Profile',
+  },
+});
+
+interface Props {
+  user: User;
+}
+
+const UserMeta = ({ user: { walletAddress, profile }, user }: Props) => {
+  const { displayName, bio, website, location } = profile || {};
+  const { user: currentUser } = useAppContext();
+
+  const isMobile = useMediaQuery({ query: query.query700 });
+  return (
+    <div className={styles.main}>
+      <div data-test="userProfileAvatar">
+        <UserAvatar
+          className={styles.avatar}
+          address={walletAddress || ''}
+          notSet={!walletAddress}
+          size={isMobile ? 'm' : 'xl'}
+          user={user}
+        />
+      </div>
+      <div className={styles.headingContainer}>
+        {displayName && (
+          <Heading
+            appearance={{ margin: 'none', size: 'medium', theme: 'dark' }}
+            text={displayName}
+            data-test="userProfileName"
+          />
+        )}
+        {currentUser?.walletAddress === walletAddress && (
```

Comment:
I'd still assign this to a variable just to make it more readable _(even though, I agree, it's a bit of waste if we use it just for one instance)_